src/lxc/conf.h

```diff
@@ -365,6 +368,13 @@ struct lxc_conf {
 
 	/* procs */
 	struct lxc_list procs;
+
+	struct lxc_shmount {
```

Comment:
adding 'lxc_' prefix isn't helpful since this is in lxc codebase.  How about shmount_path or something?